### PR TITLE
Dockerization

### LINKED ISSUE #137: Dockerization
**Describe the task**
For the AWS deployment plan, we need to dockerize the current app.

**Expected behavior**
1. Make the app dockerized
2. Update installation instructions

### PR DESCRIPTION
**Pull request by rathorevaibhav**
Fixes #137

## Summary

1. Dockerization is in place
2. Removed all major configs from zappa settings. All things moved to `.env`.
3. Entrypoint.sh to load data from Django fixture to create default tenant. Users won't have to set up default tenant anymore.
4. Updated installation instructions

## Test Plan

Nothing needed right now.

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
 
-# start server
-python manage.py migrate && python manage.py runserver 0.0.0.0:${APP_PORT}
+# run migrations
+python manage.py migrate
 
+if [ ${APP_ENV} = "local" ]; then
+    python manage.py loaddata organizations/fixtures/default_tenant.yaml
+fi
 # create default tenant
```
Comment (dalmia) [inline entrypoint.sh:10]: this documentation line should be before the code right?

File: entrypoint.sh [hunk 2]
```diff
@@ -1,11 +1,15 @@
...
 set -e
 
-# start server
-python manage.py migrate && python manage.py runserver 0.0.0.0:${APP_PORT}
+# run migrations
+python manage.py migrate
 
+if [ ${APP_ENV} = "local" ]; then
+    python manage.py loaddata organizations/fixtures/default_tenant.yaml
+fi
```
Comment (dalmia) [inline entrypoint.sh:9]: wouldn't we have to set up the public schema for staging and prod too?

File: .env.example [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# django
+APP_ENV='local' # local, staging, production
+SECRET_KEY= # generate from https://djecrety.ir/
+DEBUG=False
```
Comment (dalmia) [inline .env.example:4]: by default should this be True?

Comment (rathorevaibhav) [reply .env.example:4]: I think it depends on the user. having a different variable other than APP_ENV gives the flexibility to have any setting in place.

We can of course add the logic to make it default to True if app_env is local in settings.py. But I see that's more or less redundant.

File: entrypoint.sh (hunk 2 above)
Comment (rathorevaibhav) [reply entrypoint.sh:9]: yea we'll have to, I'm just thinking how that can happen. Given we only need public schema once and if we add a fixture with hardcoded domain, it only makes sense for Plio. Anyone else cloning this repo and setting up for themselves won't need those specific configs.

Let me work on this separately. #140

**Review by dalmia — APPROVED** [review]

**Pull request merged**